# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1456894900 18000
#      Wed Mar 02 00:01:40 2016 -0500
# Node ID 4f994c3f88fcf8a363550760c06ccae69f15bdf7
# Parent  64efef931bd05ded6ab4fe8bbb4907fbb3a185a9
don't include run-octave in both BUILT_SOURCES and ALL_LOCAL_TARGETS

* Makefile.am (DIST_INFO_FILES): Delete unused variable.
(ALL_LOCAL_TARGETS): Don't include $(noinst_SCRIPTS) in the list.
(OCTAVE_INTERPRETER_TARGETS): Don't add run-octave to the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -161,18 +161,16 @@ BUILT_SOURCES += \
 if AMCOND_HAVE_BROKEN_STL_ALGO_H
   BUILT_SOURCES += bits/stl_algo.h
 else
   BUILT_SOURCES += nonexistent-file
 endif
 
 noinst_SCRIPTS = run-octave
 
-OCTAVE_INTERPRETER_TARGETS += run-octave
-
 CLEANFILES += \
   $(BUILT_SOURCES)
 
 DISTCLEANFILES += \
   .gdbinit \
   $(DIRSTAMP_FILES)
 
 MAINTAINERCLEANFILES += \
@@ -181,18 +179,16 @@ MAINTAINERCLEANFILES += \
 
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 octinclude_HEADERS += oct-conf-post.h
 nodist_octinclude_HEADERS += config.h octave-config.h
 
 ALL_LOCAL_TARGETS += \
   $(OCTAVE_INTERPRETER_TARGETS) \
-  $(noinst_SCRIPTS) \
-  $(DIST_INFO_FILES) \
   .gdbinit \
   $(DOC_TARGETS)
 
 all-local: $(ALL_LOCAL_TARGETS)
 	@echo ""
 	@echo "Octave successfully built.  Now choose from the following:"
 	@echo ""
 	@echo "   ./run-octave    - to run in place to test before installing"
